### PR TITLE
Make site extension not use SemVer 2.0

### PR DESCRIPTION
**Pull request by jkotalik**
Still a work in process. There are some oddities in how versions numbers flow, so my attempt at this locally may not work.

I'm building this locally at the moment. Builds are slow so multiplexing verification.

### CODE REVIEW

**Review by analogrelay — COMMENTED** [review]
Hacky AF, but if it works, I'm for it :)

**Review by bradygaster — APPROVED** [review]
what @anurse said. if it works, it works, but let's hope they fix the kudu bug.

**Comment by jkotalik** [comment]
Aight, this is now actually working.

File: src/SiteExtensions/Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<Project>
```
Comment (dougbu) [inline src/SiteExtensions/Directory.Build.props]: What's the point of this file now?

File: src/SiteExtensions/LoggingBranch/LB.csproj [hunk 2]
```diff
@@ -10,17 +10,14 @@
     <PackageId>Microsoft.AspNetCore.AzureAppServices.SiteExtension.$(AspNetCoreMajorMinorVersion).$(TargetArchitecture)</PackageId>
     <RuntimeIdentifier>$(TargetRuntimeIdentifier)</RuntimeIdentifier>
     <HostingStartupRuntimeFrameworkVersion>$(MicrosoftNETCoreAppRuntimeVersion)</HostingStartupRuntimeFrameworkVersion>
-
-    <GenerateDocumentationFile>false</GenerateDocumentationFile>
```
Comment (dougbu) [inline src/SiteExtensions/LoggingBranch/LB.csproj]: Why enable generation of a doc file?

**Review by dougbu — APPROVED** [review]
Looks good but would be good to clean this up a touch

File: src/SiteExtensions/LoggingBranch/LB.csproj (hunk 2 above)
Comment (jkotalik) [reply src/SiteExtensions/LoggingBranch/LB.csproj]: Repeated attribute.

**Comment by jkotalik** [comment]
Running an internal build as PR version don't have semverv2.

**Comment by bradygaster** [comment]
if i can help test/validate let me know.

**Comment by jkotalik** [comment]
The versions look correct to me here: https://dev.azure.com/dnceng/internal/_build/results?buildId=242559&view=results. @bradygaster can you validate this?

**Pull request merged**